inc/namespace.php

```diff
@@ -19,15 +20,56 @@ function bootstrap() {
 function on_plugins_loaded() {
 	$config = get_config()['modules']['dev-tools'];
 
-	if ( $config['query-monitor'] ) {
-		// In Cloud environments, disable the Query Monitor error handler
-		// as this will override our own error handler in Cloud.
-		if ( in_array( get_environment_architecture(), [ 'ec2', 'ecs' ], true ) ) {
-			define( 'QM_DISABLE_ERROR_HANDLER', true );
-		}
+	if ( ! $config['query-monitor'] ) {
+		return;
+	}
+
+	// In Cloud environments, disable the Query Monitor error handler
+	// as this will override our own error handler in Cloud.
+	if ( in_array( get_environment_architecture(), [ 'ec2', 'ecs' ], true ) ) {
+		define( 'QM_DISABLE_ERROR_HANDLER', true );
+	}
+
+	// Hide the db.php dropin installation warning and prompt.
+	add_filter( 'qm/show_extended_query_prompt', '__return_false' );
+	require_once ROOT_DIR . '/vendor/johnbillion/query-monitor/query-monitor.php';
+
+	// Add Altis Config info to the Query Monitor.
+	// Register collector.
+	add_filter( 'qm/collectors', __NAMESPACE__ . '\\register_qm_collector_altis_config' );
+
+	// Register outputter.
+	add_filter( 'qm/outputter/html', __NAMESPACE__ . '\\register_qm_output_html_altis_config', 120, 2 );
+}
+
+/**
+ * Add Query Monitor collector for Altis Config info.
+ *
+ * @param array $collectors Array of Query Monitor collectors.
+ *
+ * @return array Array of Query Monitor collectors with added one for Altis Config.
+ */
+function register_qm_collector_altis_config( array $collectors ) : array {
+	require_once __DIR__ . '/altis-config/class-qm-collector-altis-config.php';
```

Comment:
For these requires we can add them to the `composer.json` instead by adding the following to the `autoload` property:

```json
    "classmap": [
      "inc/"
    ]
```